README.md

```diff
@@ -47,7 +47,16 @@ cargo run --example client -- hsts.badssl.com
 See [examples/server.rs](examples/server.rs). You can run it with:
 
 ```sh
-cargo run --example server -- 127.0.0.1:8000 --cert mycert.der --key mykey.der
+cargo run --example server -- 127.0.0.1:8000 --cert mycert.pem --key mykey.pem
+```
+
+If you don't have a certificate and key, you can generate a random key and
+self-signed certificate for testing with:
+
+```sh
+openssl genpkey -algorithm RSA -out mykey.pem
+openssl req -new -key mykey.pem -out csr.pem -subj /
+openssl x509 -req -in csr.pem -signkey mykey.pem -out mycert.pem
```

Comment:
For me this doesn't work (the server fails to start with `Error: InvalidCertificate(Other(OtherError(UnsupportedCertVersion)))`)

Instead let's do:

```suggestion
openssl req -newkey rsa:2048 -nodes -keyout mykey.pem \
    -x509 -addext "subjectAltName=DNS:localhost" -subj / \
    -out mycert.pem
```

Reply:
Alternatively, we could do:

```suggestion
cargo install --locked rustls-cert-gen
rustls-cert-gen --output certs/ --san localhost
```

Reply:
I like the second option since it doesn't require `openssl` to be installed. I've updated the `README.md`, including the file paths in the command invocation to run the example server. I also added the `certs` directory to `.gitignore` so you don't get anyone accidentally checking in test certificates.